site/docs/latest/deployment/kubernetes.md

```diff
@@ -1,4 +1,111 @@
 ---
```

Comment:
It might be worth adding a section on how to run an example or use `bkshell` to verify the installation.